api/v1alpha1/lvmcluster_types.go

```diff
@@ -45,6 +45,10 @@ type DeviceClass struct {
 	// +optional
 	NodeSelector *corev1.NodeSelector `json:"nodeSelector,omitempty"`
 
+	// Tolerations to apply to nodes to act on
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

Comment:
Why have this in the device class instead of the LVMCluster.Spec?

Reply:
It should go along with the NodeSelector.  Since we have nodeSelector in the DeviceClass, that's why Tolerations also in DeviceClass.